src/main/java/com/eleks/auctioneer/controller/UserController.java

```diff
@@ -0,0 +1,27 @@
+package com.eleks.auctioneer.controller;
+
+import com.eleks.auctioneer.DTO.AppUserDTO;
+import com.eleks.auctioneer.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
```

Comment:
Please rewrite URI parts according to the REST principles https://www.freecodecamp.org/news/rest-api-best-practices-rest-endpoint-design-examples/